docs/Growing.md

```diff
@@ -117,6 +117,143 @@ functionality and then moves on to producing code that passes these tests. There
 having good test coverage and code being designed for testability, which will in turn often prevent future changes from
 breaking old code. Much of mojo.js was developed using TDD.
 
+## Prototype
+
+Mojolicious allows you to both create a lite app and a full app. The lite app
+is perfect for rapid prototyping.
+
+You can create the lite app by:
+
+  $ npx mojo create-lite-app          !!!!!VERIFY THIS
```

Comment:
You forgot debug comments.

Reply:
Can you expand on this? I am not sure I understand. But, I am removing the !!!VERIFY THIS part.